guides/hosting/infrastructure/reverse-http-cache.md

```diff
@@ -195,8 +195,12 @@ For manual deployment, you can find the VCL Snippets here:
 
 The Reverse Proxy Cache shares the same invalidation mechanism as the Object Cache and has the same tags. So, when a product is invalidated, the object cache and the HTTP cache will also be invalidated.
 
-::: warning
-`bin/console cache:clear` will also clear the HTTP cache. If this is not intended, you should manually delete the `var/cache` folder. The object cache can be cleared with `bin/console cache:pool:clear --all` explicitly.
-:::
+There are a few different cache clearing commands:
+
+1. `bin/console cache:clear` - Clears and warms up the application cache
```

Comment:
we need an info when these commands exists

Reply:
yes

Reply:
@shyim can you recheck? It's not confusing at all 🙈

Reply:
Not sure if we should add `bin/console cache:clear:http `to the newest version, since it was removed already before that. It reads a little odd. Why add it to 6.7 ?

Reply:
Wdyt about a table ?

![image](https://github.com/user-attachments/assets/20fd842e-8e24-4d06-b256-6a55a233e38c)